.github/workflows/flowzone.yml

```
@@ -530,7 +530,12 @@ jobs:
           [ -f package.json ] && versions+=($(jq -r .version package.json))
           [ -f balena.yml ] && versions+=($(yq e '.version' balena.yml))
           [ -f VERSION ] && versions+=($(head -n1 VERSION))
-          semver="$(npx -q -y -- semver "${versions[@]}" | tail -n1)"
+          npm install -g semver > /dev/null 2>&1
+          semver="$(node <<- EOF
+          const semverParse = require('semver/functions/parse')
+          console.log(semverParse("${versions[0]}")['raw'])
+          EOF
+          )"
```

What's the difference between this and just doing
```suggestion
          semver="${versions[0]}"
```
?

---

My thinking was that there is an implicit semver validity check in using the semver module @Page- If that's not relevant why would it have been not done like that initially?

---

Redid the PR to the simpler version and amended the commit with reasoning.